src/app/test-record/test-section/edit/test-section-edit.component.html

```diff
@@ -36,7 +36,215 @@
+        type="text"
+      />
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.expiryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Expiry Date
+      </label>
+      <vtm-date-input
+        id="test-testExpiryDate"
+        ariaDescribedBy="testExpiryDate"
+        formControlName="testExpiryDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.anniversaryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Anniversary Date
+      </label>
+      <vtm-date-input
+        id="test-testAnniversaryDate"
+        ariaDescribedBy="testAnniversaryDate"
+        formControlName="testAnniversaryDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Test date
+      </label>
+      <vtm-date-input
+        id="test-testTypeEndTimestampDate"
+        ariaDescribedBy="testTypeEndTimestampDate"
+        formControlName="testTypeEndTimestampDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Start time
+      </label>
+      <vtm-time-input
+        id="test-testTypeStartTimestamp"
+        ariaDescribedBy="testTypeStartTimestamp"
+        formControlName="testTypeStartTimestamp"
+        ngDefaultControl
+      >
+      </vtm-time-input>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        End time
+      </label>
+      <vtm-time-input
+        id="test-testTypeEndTimestampTime"
+        ariaDescribedBy="testTypeEndTimestampTime"
+        formControlName="testTypeEndTimestampTime"
+        ngDefaultControl
+      >
+      </vtm-time-input>
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.defectsApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-full">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Prohibition Issued
+      </label>
+      <div class="govuk-radios">
+        <div class="govuk-radios__item" *ngFor="let option of prohibitionOptions">
+          <input
+            class="govuk-radios__input"
+            id="test-prohibitionIssued-{{ option.id }}"
```

Comment:
rename edit with test-edit.
We use test- for view screens and test-edit- for edit screens